deployments/gcp/multi-region/terraform.tfvars.sample

```diff
@@ -95,6 +104,9 @@ centos_admin_ssh_pub_key_file = "~/.ssh/id_rsa.pub"
 #    3. unicode characters
 # See: https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements
 
+# If using encryption, enter the KMS key ID. Otherwise, comment out.
+# kms_cryptokey_id = "projects/<project-id>/locations/<location>/keyRings/<keyring-name>/cryptoKeys/<key-name>"
```

Comment:
Same comment as above. "If using encryption, enter the KMS key ID and uncomment the line" would be more specific.